_component_design/chips.md

```diff
@@ -13,32 +13,87 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
+## Introduction
+
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Basic"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/chips.svg){:width="100%"}
-{% endfigure %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- Chips are best used with indicating a list of items that may be part of a different workflow. For example, requestors on tickets can be added and removed or management tasks that can be performed by requestors. Devices also offer a good option for chip use. 
```

Comment:
When you say "Devices," do you mean mobile devices? Laptops are a device, too.

Reply:
I removed this sentence, see above note.